src/main/java/com/google/api/generator/gapic/composer/grpc/MockServiceClassComposer.java

```diff
@@ -69,7 +69,7 @@ public GapicClass generate(GapicContext ignored, Service service) {
             .setStatements(createFieldDeclarations(service, typeStore))
             .setMethods(createClassMethods(service, typeStore))
             .build();
-    return GapicClass.create(kind, classDef);
+    return GapicClass.create(kind, classDef).withDefaultHost(service.defaultHost());
```

Comment:
I feel like we might be adding `defaulHost` to too many composers, I'm pretty sure we don't need it here and `MockServiceImplClassComposer`, probably not in `ResourceNameHelperClassComposer.java` as well, can you try to run all the tests without adding defaultHost to these composers? It will make the tests much easier

Reply:
That makes sense. Confirmed that removing `withDefaultHost` from the builders of `MockServiceClassComposer`, `MockServiceImplClassComposer`, and `ResourceNameHelperClassComposer` is fine as those composers are not used in generating samples. Have removed from builders as well as the unit tests.